spec/models/note_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Note, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
I'd add basic specs for associations and validations, as similar to the other models.